egs2/mucs21_subtask2/asr1/conf/specaug.yaml

```diff
@@ -0,0 +1,16 @@
+process:
```

Comment:
ditto.
espnet2 usually put the specaug parameters in the training config file.
Please do it, and run the experiment, again.